javascript/javascript2/week1/danishLettersInAString.js

```diff
@@ -0,0 +1,20 @@
+const danishString = ["Jeg har en blå bil"];
+const danshString1 = danishString.toString();
+const danishString2 = ["Blå grød med rød bær"];
+const danshString3 = danishString2.toString();
+function char_count(str) {
+  letter = ["æ", "ø", "å"];
```

Comment:
You could simplify the this code with an Javascript object instead: `let danishCharacters = {æ:0, ø :0, å :0}` where the key is the danish character and the value is the number of encounters in the string.